easybuild/easyconfigs/c/Clang/Clang-3.7.1-foss-2016a.eb

```diff
@@ -0,0 +1,49 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+#
+# Copyright:: Copyright 2013-2015 Dmitri Gribenko, Ward Poelmans
+# Authors:: Dmitri Gribenko <gribozavr@gmail.com>
+# Authors:: Ward Poelmans <wpoely86@gmail.com>
+# License:: GPLv2 or later, MIT, three-clause BSD.
+# $Id$
+##
+
+name = "Clang"
+version = "3.7.1"
+
+homepage = "http://clang.llvm.org/"
+description = """C, C++, Objective-C compiler, based on LLVM.  Does not
+ include C++ standard library -- use libstdc++ from GCC."""
+
+# Clang also depends on libstdc++ during runtime, but this dependency is
+# already specified as the toolchain.
+toolchain = {'name': 'foss', 'version': '2016a'}
+# Do not set optarch to True: it will cause the build to fail
+toolchainopts = {'optarch': False}
+
+source_urls = ["http://llvm.org/releases/%(version)s"]
+sources = [
+    "llvm-%(version)s.src.tar.xz",
+    "cfe-%(version)s.src.tar.xz",
+    "compiler-rt-%(version)s.src.tar.xz",
+    "polly-%(version)s.src.tar.xz",
+]
+
+dependencies = [
+    ('GMP', '6.1.0'),
+    ('ISL', '0.15'),
```

Comment:
ISL 0.15 dates from 201506, there's an ISL 0.17 already, why not use that?

similar for GMP: 6.1.1